src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,871 @@
+        """Create a new LocalStack instance.
+
+        Args:
+            name: The name of the stack, specifically the name of the folder
+                  on git
+            path: Path at which the stack is installed
+        """
+        self.name = name
+        self.path = path
+
+    @property
+    def recipes_deploy_bash_script(self) -> str:
+        """Path to the bash script that deploys the recipe.
+
+        Returns:
+            Path to the bash script that deploys the recipe.
+        """
+        return os.path.join(self.path, RECIPE_RUN_SCRIPT)
+
+    @property
+    def recipes_destroy_bash_script(self) -> str:
+        """Path to the bash script that destroys the recipe.
+
+        Returns:
+            Path to the bash script that destroys the recipe.
+        """
+        return os.path.join(self.path, RECIPE_DESTROY_SCRIPT)
+
+    # define the property here
+    @property
+    def stack_yaml_file(self) -> str:
+        """Path of the stack config YAML file created as part of recipe deployment.
+
+        Returns:
+            Path of the stack config YAML file created as part of recipe deployment.
+        """
+        stack_info_file = open(self.path / STACK_INFO_FILE, "r")
+        file_name = stack_info_file.read().replace('"', "")
+        file_name = file_name[2:]
+        stack_info_file.close()
+
+        return os.path.join(self.path, file_name.replace("\n", ""))
+
+    def is_present(self) -> bool:
+        """Checks if the stack_recipe exists at the given path.
+
+        Returns:
+            True if the stack_recipe exists at the given path, else False.
+        """
+        return fileio.isdir(str(self.path))
+
+    def run_stack_recipe(self, stack_recipe_runner: List[str]) -> None:
+        """Run the local stack recipe using the bash script at the supplied location.
+
+        Args:
+            stack_recipe_runner: Sequence of locations of executable file(s)
+                            to run the stack_recipe
+
+        Raises:
+            NotImplementedError: If the stack_recipe hasn't been implement yet.
+            FileNotFoundError: If the stack_recipe runner script is not found.
+            subprocess.CalledProcessError: If the stack_recipe runner script fails.
+        """
+        if all(map(fileio.exists, stack_recipe_runner)):
+            call = stack_recipe_runner
+            try:
+                subprocess.check_call(
+                    call,
+                    cwd=str(self.path),
+                    shell=click._compat.WIN,
+                    env=os.environ.copy(),
+                )
+            except RuntimeError:
+                raise NotImplementedError(
+                    "Please raise an issue at "
+                    "https://github.com/zenml-io/mlops-stacks"
+                )
+            except subprocess.CalledProcessError as e:
+                if e.returncode == 38:
+                    raise NotImplementedError(
+                        "Please raise an issue at "
```

Comment:
Its not implemented so raising an issue should not be the behavior. Maybe rather point them to the hellonext?

Reply:
This not implemented error points to the absence of a `run_recipe` script within an already existing stack recipe. So, I feel an issue should be created to investigate why the script isn't implemented or why it couldn't be located by the CLI code. 
Raising a feature request on hellonext might not be ideal since a stack recipe already exists and only a modification/fix is needed.

Reply:
True. Moreover, I think a not implemented error here doesn't make a lot of sense too because the run command is called on an existing stack recipe so it should exist. I'll remove the `NotImplementedError` and just keep the `CalledProcessError`.